plugins/apps/threatbus-misp/threatbus_misp.py

```diff
@@ -61,6 +61,33 @@ def publish_sightings(outq):
         lock.release()
 
 
+def forward_raw(msg_str, inq):
+    """Gracefully tries to parse and forward a message string. If the message
+        cannot be parsed, an error is logged and None is returned.
+        @param msg_str The message (JSON string) to forward
+        @param inq The queue to forward successfully parsed message to
+    """
+    global logger
+    try:
+        msg = json.loads(msg_str)
+    except Exception as e:
+        logger.error(f"Error decoding message {msg_str}: {e}")
+        return
+    all_intel = []
+    if msg.get("Attribute", None):
+        intel = map_to_internal(msg["Attribute"], msg.get("action", None), logger)
+        if intel:
+            all_intel.append(intel)
+    elif msg.get("Event", None) and msg.get("action", None) == "delete":
```

Comment:
I understand that you are preparing for https://github.com/MISP/MISP/issues/4450 here. While you're at it, how about logging actions other than `delete`.

When we add an event, I think we don't have to do anything because we get all the contained attributes as well. Is that correct?

Then the only other action left is updating an event. I haven't check this, but I would assume that event updates don't matter for Threat Bus right now, because at this point we only care about attribute updates.